### PR TITLE
cmd: add db import export

### PR DESCRIPTION
**Pull request by acud**
Adding Import/Export functionality to `bee`

### CODE REVIEW

**Comment by Eknir** [comment]
Why do we need this?

File: cmd/bee/cmd/db.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+			dataDir, err := cmd.Flags().GetString(optionNameDataDir)
+			if err != nil {
+				return fmt.Errorf("get data-dir: %v", err)
+			}
+			if dataDir == "" {
+				return errors.New("no data-dir provided")
+			}
+
+			fmt.Printf("starting export process with data-dir at %s\n", dataDir)
```
Comment (janos) [inline cmd/bee/cmd/db.go:57]: Maybe to use logger here instead the print? Mainly for having the timestamps here and on the line where the export ends. WDYT?

File: cmd/bee/cmd/db.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+			}
+			dataDir, err := cmd.Flags().GetString(optionNameDataDir)
+			if err != nil {
+				return fmt.Errorf("get data-dir: %v", err)
+			}
+			if dataDir == "" {
+				return errors.New("no data-dir provided")
+			}
+
+			fmt.Printf("starting import process with data-dir at %s\n", dataDir)
```
Comment (janos) [inline cmd/bee/cmd/db.go:117]: The same question about logger...

**Review by janos — APPROVED** [review]
👍

**Review by zelig — APPROVED** [review]

**Review by aloknerurkar — APPROVED** [review]

**Pull request merged**